client/run.go

```diff
@@ -254,6 +258,15 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 		}
 	}
 
+	for _, v := range cmdVols {
+		vol, volRef, err := parseVolume(v)
+		if err != nil {
+			return "", err
+		}
+		volumes = append(volumes, *vol)
+		volumesRef = append(volumesRef, *volRef)
```

Comment:
I prefer to generate the volume name here. 
fmt.Sprintf("vol-%d", i); `i` is the order of the volumes.